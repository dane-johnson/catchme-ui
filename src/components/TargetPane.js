import React from 'react'
import { Well } from 'react-bootstrap'

const style = {
  userImage: {
    display: 'inline-block'
  },
  userInfo: {
    marginLeft: '70%',
    display: 'inline-block'
  }
}

const TargetPane = ({target}) => {
  if (!target) return null;
  return (
    <Well>
      <img alt="User" style={style.userImage}></img>
      <span style={style.userInfo}>Name: {target.name}</span>
    </Well>
  )
}

export default TargetPane
