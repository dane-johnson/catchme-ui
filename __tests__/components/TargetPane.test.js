import React from 'react'
import renderer from 'react-test-renderer'
import TargetPane from '../../src/components/TargetPane'

const mockTarget = {
  name: 'Frank'
}

describe('<TargetPane />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <TargetPane target={mockTarget} />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
