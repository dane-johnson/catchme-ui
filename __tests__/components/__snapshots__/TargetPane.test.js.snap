exports[`<TargetPane /> renders correctly 1`] = `
<div
  className="well">
  <img
    alt="User"
    style={
      Object {
        "display": "inline-block",
      }
    } />
  <span
    style={
      Object {
        "display": "inline-block",
        "marginLeft": "70%",
      }
    }>
    Name: 
    Frank
  </span>
</div>
`;
